src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -708,7 +708,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_buffered(int vni, struct fi_cq_tagged
 {
     int rc = 0;
```

Comment:
If we do this, we should rename `rc` into something meaningful.

Reply:
done